pageserver/src/tenant.rs

```diff
@@ -1079,12 +1084,13 @@ impl Tenant {
     /// This is used to create the initial 'main' timeline during bootstrapping,
     /// or when importing a new base backup. The caller is expected to load an
     /// initial image of the datadir to the new timeline after this.
-    pub fn create_empty_timeline(
-        &self,
+    pub fn create_empty_timeline<'a>(
+        self: &'a Arc<Self>,
```

Comment:
```suggestion
        &self,
```

Neither of the inside methods need `Arc`?